code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -388,9 +388,11 @@
 		msg += "<span class = 'deptradio'>Physical status:</span> <a href='?src=[UID()];medical=1'>\[[medical]\]</a>\n"
 		msg += "<span class = 'deptradio'>Medical records:</span> <a href='?src=[UID()];medrecord=`'>\[View\]</a> <a href='?src=[UID()];medrecordadd=`'>\[Add comment\]</a>\n"
 
-
 	if(print_flavor_text() && !skipface)
-		msg += "[print_flavor_text()]\n"
+		if(get_organ("head"))
+			var/obj/item/organ/external/head/H = get_organ("head")
+			if(!H.disfigured)
```

Comment:
i happened to get more confused that I should have been reading this, and assumed that H can be `null` (which it cannot). I'd prefer something like
```suggestion
		var/obj/item/organ/external/head/H = get_organ("head")
		if(H && !H.disfigured)
```
(One can even say that this is "faster", because it's only one call to `get_organ` instead of two, but that's honestly irrelevant)